docs/i18n/fr/docusaurus-plugin-content-docs/current/getting-started/demo/with-docker.md

```diff
@@ -0,0 +1,83 @@
+
+:::caution Avertissement!
+Le noeud Cardano créé un "réseau" privé et local qui démarre depuis un bloc genesis. Pour démarrer, la configuration du noeud doit être à jour (comprendre, récente). Si au démarrage, le noeud Cardano retourne `LedgerNoView`, alors c'est que l'heure de démarrage indiquée dans la configuration génésis est trop loin dans le passé: il faut en ce cas générer la configuration à nouveau via `prepare-devnet.sh`.
+:::
+
+## Mise en place du réseau
+
+Nous utiliserons [Docker](https://www.docker.com/get-started) et [compose](https://www.docker.com/get-started) pour cette démo, assurez-vous de les avoir accessibles. Si vous ne souhaitez pas utiliser / installer Docker, vous pouvez directement vous rendre sur [Démo: Sans Docker](/docs/getting-started/demo/without-docker) et mettre les mains dans le cambouis.
+
+:::info Contexte
+Toutes les commandes ci-dessous supposent qu'elles sont éxécutées depuis le dossier `demo/`, à la racine du dépôt. Vous aurez donc besoin de cloner le dépôt et de `cd demo` avant d'aller plus loin.
+:::
+
+:::warning (in)compatibilité du système d'exploitation
+Les instructions fournies ci-après ont été testées pour des environnements Linux récents (Ubuntu, ArchLinux, NixOS...). Si vous êtes sur Windows ou MacOS vous aurez sans doute besoin d'ajuster certaines commandes liées aux [volumes](https://docs.docker.com/storage/volumes/).
+:::
+
+Pour commencer, récupérez les images des différents services définis dans le fichier compose:
+
+```mdx-code-block
+<TerminalWindow>
+docker-compose --profile tui pull
+</TerminalWindow>
+```
+
+D'ici, vous pouvez exécuter le script `./prepare-devnet.sh` pour créer les fichiers nécessaires à la configuration du réseau local (a.k.a devnet). Le script génère une configuration génésis Cardano (qui fixe les paramètres du protocole). Notez que dans le cadre de cette démo, la configuration du réseau n'utilise pas de stake pool et ne requiert qu'un unique noeud pour fonctionner.
+
+```mdx-code-block
+<TerminalWindow>
+./prepare-devnet.sh
+</TerminalWindow>
+```
+
+C'est tout. Il est maintenant possible de démarrer le réseau via:
+
+```mdx-code-block
+<TerminalWindow>
+docker-compose up -d
+</TerminalWindow>
+```
+
+Toutes ces étapes peuvent être effectuées sont combinées dans un unique script `./run-docker.sh`. Ce script contient aussi quelques petites vérifications pour éviter de vous évitez de vous tirer une balle dans le pied avec certaines commandes. 
+
+## Générer des fonds
+
+En l'état actuel, les noeuds Hydra requièrent des UTxO dans une forme assez spécifique afin de pouvoir payer les transactions nécessaires au protocole (on appelle ces UTxO: "carburant"), ainsi que des fonds à bloquer dans le contrat pour chaque participant. Parmi les fichiers, vous trouverez un script `./seed-devnet.sh` qui s'occupe de générer ces UTxOs pour chaque participant en utilisant la `cardano-cli`. 
```

Comment:
Suggestion: `bloquer` -> `consigner` ? _commit_ is hard to translater, I kind of like the semantic associations around `consigner`